modules/govuk/manifests/apps/content_store.pp

```diff
@@ -89,6 +93,13 @@
     }
   }
 
+  if $unicorn_worker_processes {
+    govuk::app::envvar { "${title}-UNICORN_WORKER_PROCESSES":
```

Comment:
In #7203 I added this to govuk::app, so you can replace this stanza with passing `$unicorn_worker_processes` into the govuk::app call on line68 above.

Reply:
Oh excellent. I'll get that sorted out.